src/lightning_app/core/flow.py

```diff
@@ -230,6 +230,12 @@ def __getattr__(self, item):
             return Path.from_dict(self._paths[item])
         return self.__getattribute__(item)
 
+    @property
+    def ready(self) -> bool:
+        """Override to customize when your App should be ready."""
```

Comment:
The user doesn't know what it means for it to be ready. The impact of this is that the button on the admin UI becomes enabled and the user can open the UI when the app is ready. I think we should explicitly say that here. 

Also, as discussed offline, the behavior here is special. The ready property will only be re-evaluated until it turns to True and then never again. Meaning the state here can't change arbitrarily during runtime. This probably makes sense, so I suggest to document the behavior.

Reply:
This would be clarified once local dev is added properly.